client/app/bundles/course/assessment/submission/components/ScribingView/ScribingToolbar.jsx

```diff
@@ -391,7 +379,10 @@ class ScribingToolbar extends Component {
             lineToolType={scribingToolThickness.SHAPE_BORDER}
             open={this.state.popovers[scribingPopoverTypes.SHAPE]}
             anchorEl={this.state.popoverAnchor}
-            onRequestClose={() => (this.onRequestClosePopover(scribingPopoverTypes.SHAPE))}
+            onRequestClose={() => {
+              this.onRequestClosePopover(scribingPopoverTypes.SHAPE);
+              this.props.setNoFill(this.props.answerId, false);
```

Comment:
Might be good to leave this out first since it may lead to unintended behavior when we make objects editable

Reply:
The `this.props.setNoFill(this.props.answerId, false);` is meant to clear the tick in the checkbox in the redux store. It doesn't have any side effects when I made objects editable on the same branch. (I don't maintain state of whether an object has 'no fill' ticked or not, it's only a shortcut to set alpha to 0)

Reply:
Ah okay. Can leave it then.